File: backend/src/main/java/com/Fijalkowskim/SpaceFarmManagmentSystem/requestmodels/ControlRequest.java
package com.Fijalkowskim.SpaceFarmManagmentSystem.requestmodels;

import com.Fijalkowskim.SpaceFarmManagmentSystem.models.Reading;
import lombok.Data;

import java.util.Date;
import java.util.Set;

@Data
public class ControlRequest {
    private Date controlDate;

    private int deadSeedlings;

    private Set<Reading> readings;
}

File: frontend/src/models/requestmodels/ControlRequest.js
export class ControlRequest {
  constructor(controlDate = new Date(), deadSeedlings = 0, readings, stageId) {
    this.controlDate = controlDate;
    this.deadSeedlings = deadSeedlings;
    this.readings = readings;
    this.stageId = stageId;
  }
}
